osu.Game/Users/Drawables/UpdateableAvatar.cs

```diff
@@ -44,20 +44,20 @@ public new EdgeEffectParameters EdgeEffect
 
         protected override double LoadDelay => 200;
 
-        private readonly bool openOnClick;
+        private readonly bool isInteractive;
         private readonly bool showUsernameTooltip;
         private readonly bool showGuestOnNull;
 
         /// <summary>
         /// Construct a new UpdateableAvatar.
         /// </summary>
         /// <param name="user">The initial user to display.</param>
-        /// <param name="openOnClick">Whether to open the user's profile when clicked.</param>
-        /// <param name="showUsernameTooltip">Whether to show the username rather than "view profile" on the tooltip.</param>
+        /// <param name="isInteractive">If set to true, hover/click sounds will play and clicking the avatar will open the user's profile.</param>
+        /// <param name="showUsernameTooltip">Whether to show the username rather than "view profile" on the tooltip. (note: this only applies if <paramref name="isInteractive"/> is also true)</param>
```

Comment:
This parameter interdependency feels kinda ehh. If this is to be the case then specifying `isInteractive: false, showUsernameTooltip: false` should probably throw `NotSupportedException` rather than silently fail...

Not that I like that the two are interdependent in the first place. This could potentially be avoided by an enumeration type (something like `NonInteractive`, `Clickable`, `ClickableWithTooltip`?) but unsure if not too much of a muchness.

Reply:
I think that's better than two booleans and an exception throw on one case yeah.

Reply:
The weird part about this is it has a tooltip in both cases. So the enum would need to be something like `ClickableWithUserTooltip` for the third case. I'm not necessarily against that, but it's a bit unfortunate (and unfortunate that we can't apply that portion with a derived class).

@smoogipoo thoughts on direction?

Reply:
I dunno. Keep it simple until necessary?

Reply:
So just an exception for now?

Reply:
With or without, I'm not fussed either way.

Reply:
Inclined to just leave as-is, I think it's obvious enough.

Reply:
I'd agree. It's not like it's an API class so I'd trust the review process to confirm proper usage/operation.